lib/c-api/build.rs

```diff
@@ -374,4 +374,9 @@ fn exclude_items_from_wasm_c_api(builder: Builder) -> Builder {
         .exclude_item("wasi_get_start_function")
         .exclude_item("wasi_get_wasi_version")
         .exclude_item("wasi_version_t")
+        .exclude_item("wasm_instance_get_vmctx_ptr")
```

Comment:
Perhaps I misunderstood how the header file generation works, but my understanding is that we wanted it to be not ignored so that it's part of `wasmer_wasm.h`.

Or are you saying that we can specify `wasmer_c_api::wasmer` in `exclude_item` and everything just works?

Reply:
This `wasm_instance_get_vmctx_ptr` is already marked as `cbindgen:ignore` right? So you don't need to exclude this item for the `wasmer.h*` files.

Reply:
Your next comment makes me realize that `wasm_instance_get_vmctx_ptr` is a public function.

Forget my comment!